### PR TITLE
feat: process_soit for floe tagging

### PR DESCRIPTION
**Pull request by cpaniaguam**
#343

### CODE REVIEW

File: src/tracker/tracker-funcs.jl [hunk 1]
```diff
@@ -497,3 +497,48 @@ function addfloemasks!(props, imgs)
...
+The SOIT output file is a csv file with columns :Date, :sat1, :sat2, ... :satn, where each row is a pass time for the satellites listed in the columns. The resulting DataFrame has columns `:sat` and `:pass_time` sorted by :pass_time.
+
+# Arguments
+- `passtimesdir::String`: The directory containing the SOIT output file
+"""
+function process_soit(passtimesdir::String)
+    # check there is a file at passtimesdir starting with "passtimes_lat" with extension .csv
+    pth = filter(x -> occursin(r"^passtimes_lat.*\.csv$", x), readdir(passtimesdir))
+    isempty(pth) && error("No csv file found at $passtimesdir starting with 'passtimes_lat'")
+    length(pth) > 1 && error("More than one csv file found at $passtimesdir starting with 'passtimes_lat'")
```
Comment (tdivoll) [inline src/tracker/tracker-funcs.jl:515]: Should we handle the multiple soit files during param cycling? The results structure looks different when expanding parameters: 
image

**Review by tdivoll — COMMENTED** [review]
The code looks good, I was just wondering if we should try to handle the multiple files differently, maybe using the date that is passed to the function?

File: src/tracker/tracker-funcs.jl (hunk 1 above)
Comment (cpaniaguam) [reply src/tracker/tracker-funcs.jl:515]: @tdivoll This is now handled by cycl. I think this is okay to merge?

**Comment by cpaniaguam** [comment]
These features will be invoked from the pipeline repo so they don't need to be moved over here.

**Pull request closed without merging**